specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2021-12-01/Marketplace.json

```diff
@@ -0,0 +1,2894 @@
+          "type": "string"
+        },
+        "offerIdsList": {
+          "description": "Offers ids list to transfer from source collection to target collection(s)",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "CollectionsToSubscriptionsMappingPayload": {
+      "description": "The subscriptions list to get the related collections",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/CollectionsToSubscriptionsMappingProperties",
+          "description": "Subscriptions ids list"
+        }
+      }
+    },
+    "CollectionsToSubscriptionsMappingProperties": {
+      "description": "The subscriptions list to get the related collections",
+      "type": "object",
+      "properties": {
+        "subscriptionIds": {
+          "description": "Subscriptions ids list",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "CollectionsToSubscriptionsMappingResponse": {
+      "description": "A map of collections subscriptions details",
+      "type": "object",
+      "properties": {
+        "details": {
+          "$ref": "#/definitions/CollectionsToSubscriptionsMappingResponseProperties",
+          "description": "The map of collections subscriptions"
+        }
+      }
+    },
+    "CollectionsToSubscriptionsMappingResponseProperties": {
+      "description": "Collections - subscriptions mapping details, map key is collectionId. In case subscriptions list is null or empty: no subscription from the list is related to that collection.",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/collectionsSubscriptionsMappingDetails"
+      }
+    },
+    "collectionsSubscriptionsMappingDetails": {
+      "description": "Collection name and related subscriptions list",
+      "type": "object",
+      "properties": {
+        "collectionName": {
+          "description": "Collection name",
+          "type": "string"
+        },
+        "subscriptions": {
+          "description": "Subscriptions ids list",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "StopSellOffersPlansNotificationsList": {
+      "description": "List of stop sell offers and plans notifications.",
+      "type": "object",
+      "properties": {
+        "stopSellNotifications": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StopSellOffersPlansNotificationsListProperties"
+          }
+        }
+      }
+    },
+    "StopSellOffersPlansNotificationsListProperties": {
+      "description": "List of stop sell offers and plans notifications.",
+      "type": "object",
+      "properties": {
+        "offerId": {
+          "description": "Gets offer id",
```

Comment:
>Gets  [](http://example.com/codeflow?start=26&length=5)

Recommend removing "Gets". These are property descriptions not actions/methods.

Reply:
fixed - removed